### PR TITLE
feat: add new lib for generate client conf and put in session storage

### PR DESCRIPTION
**Pull request by labuchTractr**

### CODE REVIEW

**Comment by maxmousse** [comment]
@labuchTractr On va attendre la validation et le retour de @floross pour merger celle là: elle est un peu touchy et elle introduit des breaking changes.

File: apps/admin/src/environments/environment.default.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { transformAndValidate } from '@trxn/common';
 export const validateEnv = transformAndValidate(AdminEnv);
 export const validateAppConfig = transformAndValidate(AppConfig);
 
-export function getConfig(appConfig: Record<string, unknown>): AppConfig {
+export function getConfig(appConfig: AppConfig): AppConfig {
```
Comment (floross) [inline apps/admin/src/environments/environment.default.ts]: I don't think it is an AppConfig that you take in params but more an AdminEnv

**Review by floross — CHANGES_REQUESTED** [review]
You got a comment on AppConfig in the admin app. Except this part all is good for me. I'll put the changes to do in the next migration

File: apps/admin/src/environments/environment.default.ts (hunk 1 above)
Comment (labuchTractr) [reply apps/admin/src/environments/environment.default.ts]: you'r right  but it's curious that TS did'nt yell at me

**Review by floross — APPROVED** [review]

**Pull request merged**